### PR TITLE
document how to configure resolved on rhel

### PR DESCRIPTION
**Pull request by booxter**

### CODE REVIEW

File: devsetup/scripts/ipv6-nat64/README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
 
+Prerequisites
+-------------
+
+Network setup relies on `systemd-resolved` service enabled and used for DNS
+resolution. If it's not installed on your machine (for example, if you use
+RHEL), then:
+
+#. Install the package: `dnf install -y systemd-resolved`
+#. Enable and start the service: `systemctl enable systemd-resolved && systemctl start systemd-resolved`
```
Comment (karelyatin) [inline devsetup/scripts/ipv6-nat64/README.md:12]: could use systemctl enable --now systemd-resolved instead to start along with enable

**Review by karelyatin — COMMENTED** [review]
/lgtm

**Comment by booxter** [comment]
```
2024-10-09 10:37:49.214327 | controller |     Error: reading blob sha256:9910d6e6de7a32916d893051073af5bc030d3f7f5fa97ae9459588245d03244f: fetching blob: received unexpected HTTP status: 502 Bad Gateway
```

**Comment by booxter** [comment]
recheck

**Review by karelyatin — COMMENTED** [review]
/lgtm

**Review by fao89 — APPROVED** [review]

**Pull request merged**